projects/swimlane/ngx-ui/src/lib/components/input/input.component.ts

```diff
@@ -250,7 +250,7 @@ export class InputComponent implements AfterViewInit, OnDestroy, ControlValueAcc
   private _required = false;
   private _autoSelect = false;
   private _autofocus = false;
-  private _autocomplete = false;
+  private _autocomplete = 'off';
```

Comment:
nitpick: adding type `'on' | 'off' | boolean` for stronger typing
```suggestion
  private _autocomplete: 'on' | 'off' = 'off';
```

Reply:
Going with `boolean | string`